pybamm/input/parameters/lithium-ion/negative_electrodes/graphite_Yang2017/plating_exchange_current_density_OKane2020.py

```diff
@@ -0,0 +1,26 @@
+from pybamm import Scalar
+
+
+def plating_exchange_current_density_OKane2020(c_e, c_Li, T):
```

Comment:
check function name?

Reply:
this function is defined twice (once in the plating parameters and once in the negative electrode parameters). This once can be removed